tensordict/tensordict.py

```diff
@@ -3248,6 +3248,46 @@ def unlock_(self) -> TensorDictBase:
 
     unlock = _renamed_inplace_method(unlock_)
 
+    def is_floating_point(self):
+        for item in self.values(True, True):
```

Comment:
why don't we use kwargs for improved readability?

Reply:
fixed